src/HomeWorkApp2.java

```diff
@@ -0,0 +1,61 @@
+public class HomeWorkApp2 {
+    public static void main(String[] args) {
+        CheckSum(2,12); // 1 задание
+        CheckPositiveNumb(-10); // 2 задание
+        BooleanCheck(2); // 3 задание
+        ReString("I love JAVA and GB", 5); // 4 задание
+        CheckLeapYear(1800); // 5 задание, делал самостоятельно, без помощи интернета
+    }
+
+    private static boolean CheckLeapYear(int i) {
+        boolean b = false;
+        if (i % 4 == 0) {
+            if (i % 100 == 0 & i % 400 == 0) {
```

Comment:
второе условие автоматом удовлетворяет первое, можно упростить
```suggestion
            if (i % 400 == 0) {
```